Automattic/wp-calypso
# Reader: Show likers only when hovering the like count, instead of the star

Context: paYKcK-3F0-p2

## Proposed Changes

* Currently, when a user hovers the like star, the popover with the Likers is shown. In my P2 post, I argue that it doesn't seem to be the appropriate behavior.
* I decided to show the Likers only upon hovering the Like count. Hovering the star will no longer show the popover.
* This will also work for the mobile version(if the user clicks on the Like count). However, I left it out because we still need to define the best way to close the popover on the mobile version.

## Testing Instructions



* Use the Calypso Live link or apply this PR to your local environment
* Navigate to `/discover`
* Test the Like button in both the comments and cards on the web version. The Likers popover should show only when hovering the Like count.
* Ensure that clicking on the star will like the post.

## Pre-merge Checklist
